pandas/tests/series/test_constructors.py

```diff
@@ -888,3 +884,14 @@ def test_constructor_generic_timestamp_deprecated(self):
         msg = "cannot convert datetimelike"
         with tm.assert_raises_regex(TypeError, msg):
             Series([], dtype='M8[ps]')
+
+    def test_constructor_overflow_coercion_signed_to_unsigned(self):
+        # GH 15832
+        for t in ['uint8', 'uint16', 'uint32', 'uint64']:
+            with pytest.raises(OverflowError):
+                Series([-1], dtype=t)
+
+    def test_constructor_overflow_coercion_float_to_int(self):
+        # GH 15832
+        with pytest.raises(OverflowError):
```

Comment:
same as above, move with related tests & test all uint & int dtypes.
assert that this works with float32/float64.